### ISSUE TITLE
[MPI_erweiterung_010]: Pandora / IsoMemo : Update Texts and Logos #216

### ISSUE THREAD

**Issue by jroachell15** [Pandora-IsoMemo/DSSM#6]
original ticket: https://github.com/INWT/mpi-iso-app/issues/216

Modelling options, and all other aspects of the app other than those listed above, are the same for both skins, when Pandora is selected the Data source remains “Database”. Any additional programming tasks listed below as part of the IsoMemo app are to be made available for both skins. Texts shown within the app for selected options currently only refer to “IsoMemo” applications (or other formulations) or in some cases to “Pandora & IsoMemo”. From now on, these texts should refer only to the selected skin option. Each skin will also have its own logo (to be provided).

Things to be done

Update texts
Update logo

**Comment by jroachell15** [comment]
@isomemo 

Do you have the logos and text that we can implement them into the app? or have we already accomplish this?

**Comment by isomemo** [comment]
@jroachell15 would it be Ok to do this later on and reserve the respective programming time? I would prefer to do this one app at a time, once I believe that this is ready to be presented to the general public.

**Comment by arunge** [comment]
@isomemo I am closing this tasks which is outdated. There is another issue regarding the update of logos:
- https://github.com/Pandora-IsoMemo/shiny-tools/issues/1
